core/src/commonMain/kotlin/behavior/GuildBehavior.kt

```diff
@@ -824,6 +825,29 @@ public suspend inline fun GuildBehavior.createNewsChannel(
     return Channel.from(data, kord) as NewsChannel
 }
 
+/**
+ * Requests to create a new stage channel.
+ *
+ * @return The created [StageChannel].
+ *
+ * @throws [RestRequestException] if something went wrong during the request.
+ */
+public suspend inline fun CategoryBehavior.createStageChannel(
```

Comment:
should have `GuildBehavior` as the receiver

Reply:
💀 Oml

Reply:
Fixed